fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -783,6 +784,14 @@ public void executeByLegacy(TUniqueId queryId) throws Exception {
         }
     }
 
+    private void syncJournalIfNeeded() throws Exception {
+        final Env env = context.getEnv();
+        if (env.isMaster() || !context.getSessionVariable().enableStrongConsistencyRead) {
+            return;
+        }
+        new MasterOpExecutor(context).execute();
```

Comment:
Not a good design to just call `execute()` here. It is confusing and unclear.
Better use a explicit method name to tell people what you are doing, eg: 
`new MasterOpExecutor(context).syncJournal();`

Reply:
And for `SyncJournalOnly` request, I think it is unnecessary to build that much parameters in `buildForwardParams()` method. It should be a very light operation.